storybook/config.js

```diff
@@ -6,11 +6,132 @@ import i18n from '@/i18n'
 
 import vocabularyTheme from './theme'
 
+// Configuration of addon-viewport
+
+const newViewports = {
+
+  // Mobile devices
+
```

Comment:
Include some more mobiles with smaller screens, such as iPhone 6S, SE and XR. Maybe a couple of other resolutions.